packages/flight-icons/catalog.json

```diff
@@ -1920,37 +1956,37 @@
       "height": 16
     },
     {
-      "id": "855:190",
+      "id": "4425:94",
       "fileName": "twitter-24",
       "iconName": "twitter",
-      "description": "twitter",
+      "description": "",
       "size": "24",
       "width": 24,
       "height": 24
     },
     {
-      "id": "855:192",
+      "id": "4425:96",
       "fileName": "twitter-16",
       "iconName": "twitter",
-      "description": "twitter",
+      "description": "",
       "size": "16",
       "width": 16,
       "height": 16
     },
     {
-      "id": "855:197",
+      "id": "4425:101",
       "fileName": "twitter-color-24",
       "iconName": "twitter-color",
-      "description": "twitter",
+      "description": "",
```

Comment:
any idea where these descriptions went?

Reply:
Not sure, would I be able to still re-input the descriptions here?

Reply:
yes, we can't edit them here

Reply:
otherwise the change here will get overwritten then next time we do an export